src/Gui/3Dconnexion/GuiNativeEventWin32.h

```diff
@@ -62,7 +62,11 @@ namespace Gui
 
     private:
         bool InitializeRawInput(HWND hwndTarget);
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
         static bool RawInputEventFilter(void* msg, long* result);
+#else
+        static bool RawInputEventFilter(void* msg, qintptr* result);
+#endif
```

Comment:
The other platforms also need to be adjusted -> GuiNativeEventLinuxX11

Reply:
This has been resolved elsewhere for the other platforms, which (as of now) do not directly include a RawInputEventFilter method.